lib/token-assert.js

```diff
@@ -110,12 +113,50 @@ TokenAssert.prototype.differentLine = function(options) {
         return;
     }
 
-    if (token.loc.end.line === nextToken.loc.start.line) {
+    var atLeast;
+    var atMost;
+    var exactly;
+
+    var useOptions = options.hasOwnProperty('atLeast') ||
+        options.hasOwnProperty('atMost') ||
+        options.hasOwnProperty('exactly');
+
+    if (useOptions) {
+        atLeast = options.atLeast;
+        atMost = options.atMost;
+        exactly = options.exactly;
+    } else {
+        atLeast = 1;
+    }
+
+    var errorMessage = token.value + ' and ' + nextToken.value + ' should be ';
+
+    var emitError = (function(message) {
         this.emit('error', {
-            message: options.message || token.value + ' and ' + nextToken.value + ' should be on different lines',
+            message: options.message || errorMessage + message + ' line(s) apart',
             line: token.loc.end.line,
             column: token.loc.end.column
         });
+    }).bind(this);
+
+    var lineDifference = Math.abs(token.loc.end.line - nextToken.loc.start.line);
+
+    if (atLeast !== undefined) {
```

Comment:
I'm not sure about atMost, atLeast, and exactly (cc @mdevils), but if it's ok — can we make a switch-case here?
Like:
```
swtich (true) {
  case !!exactly:
    error = 'exactly' + exactly;
    break;
  case !atLeast:
    error = 'at least ' + atLeast;
    break;
  case !atLeast:
    error = 'at most ' + atMost;
    break;
  default:
    error = 'more than ' + atLeast + ' and less than ' + atMost;
    break;
}
```